### ISSUE TITLE
Arduino Setup Script post build

### ISSUE THREAD

**Issue by ZachTurner07** [AESD-Course-Project/AESD-Course-Project.github.io#4]
- [x] Create a shell script to pull Arduino-ide, arduino-CLI with necessary libraries to integrate TinyML Arduino Board and shield.

DOD: The Arduino Code should be tested to build on Host Machine and tested thoroughly to download and build the TinyML libraries and tensorflow bed to setup a script to install and set under $HOME/software

Blockers:

Setup script to be tested on Tegra/Jetson post Successful Yocto build for portability test.
Build Yocto Build and test on target.

**Comment by arpit6232** [comment]
I feel this is slight duplication of the issue https://github.com/AESD-Course-Project/AESD-Course-Project.github.io/issues/6. 
Could the two be merged @ZachTurner07 . 

For testing on the Jetson. We need a successful Yocto build script.  For which we should wait.

**Comment by arpit6232** [comment]
There seems to some libraries which are build not developed for ARM devices for the Arduino-CLI. The future plan would be to transfer code on the Arduino on X86 (host). Plug and play on Jetson.

**Comment by ZachTurner07** [comment]
Does this issue describe updates to the Jetson or the host machine? I pulled this from your individual issues when I created it, and thought it was for a user to get the environment set up to interact with the Arduino. 

This seems more like an enhancement than a demo milestone, so that there's a one line setup command to be able to flash the arduino. However, if we have that done then we can for sure close it.
